Library/Formula/ffmpeg.rb

```diff
@@ -33,6 +33,8 @@ class Ffmpeg < Formula
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
   option "with-snappy", "Enable Snappy library"
   option "with-dcadec", "Enable dcadec library"
+  option "with-rubberband", "Enable rubberband library"
+  option "with-zimg", "Enable z.lib library"
```

Comment:
`zlib` is a well established compression library. Let's go with `zimg` to avoid the confusion.